client/my-sites/site-settings/wrap-settings-form.jsx

```diff
@@ -6,6 +6,7 @@ import { bindActionCreators } from 'redux';
 import { flowRight, omit, keys, pick } from 'lodash';
 import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
+import shallowEqual from 'react-pure-render/shallowEqual';
```

Comment:
I don't know that we have it tracked anywhere, but we should seek to drop `react-pure-render`. It was once useful for its `PureComponent` component before React implemented its own, but has been effectively deprecated since (["No Maintenance Intended"](https://github.com/gaearon/react-pure-render)).

Instead, we have available either [`react-addons-shallow-compare`](https://github.com/Automattic/wp-calypso/blob/3e8286efdb2b27fb8478522f0b74802fee4bea1b/package.json#L101) or [`_.isEqual`](https://lodash.com/docs/4.17.4#isEqual) (which is a deep equality check, but should be fine if you know the shapes of the objects are shallow anyways).